packages/api-client-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gadgetinc/api-client-core",
-  "version": "0.13.9",
```

Comment:
Can you do only a patch version bump instead of a minor version? I think this should be viewed as a bugfix which should just increase the patch version